src/components/Table/Table.jsx

```diff
@@ -568,7 +572,11 @@ const Table = props => {
       visibleData &&
       visibleData.length ? ( // don't show pagination row while loading
         <Pagination
-          {...view.pagination}
+          pageSize={paginationProps.pageSize}
+          pageSizes={paginationProps.pageSizes}
+          page={paginationProps.page}
+          isItemPerPageHidden={paginationProps.isItemPerPageHidden}
+          totalItems={paginationProps.totalItems < maxPages ? paginationProps.totalItems : maxPages}
```

Comment:
Is the calculation correct?   Should it be
```
totalItems={paginationProps.totalItems < (maxPages * paginationProps.pageSize) ? paginationProps.totalItems : (maxPages * paginationProps.pageSize)}
```
Is `totalItems` the total number of items or the total number of pages?

Reply:
Yeah it looks like `totalItems` should be multiplied against the `pageSize`.